index-config/src/main/solr-config/uniprot-collections/publication/conf/schema.xml

```diff
@@ -7,7 +7,7 @@
 	<field name="_version_" type="plong" indexed="true" stored="true" />
 
 	<!-- =============================== publication fields =============================== -->
-	<field name="id" type="string" indexed="true" stored="false"
+	<field name="id" type="string" indexed="true" stored="true"
 		required="true" multiValued="false" docValues="true" />
```

Comment:
What are we missing if we keep stored=false? Making this field as stored is adding more than 10GB to the publication cluster.

Reply:
I removed this warning message from logs: 13:20:17.952 [coreLoadExecutor-13-thread-3] WARN  org.apache.solr.schema.IndexSchema - uniqueKey is not stored - distributed search and MoreLikeThis will not work